code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -236,6 +236,8 @@
 				msg = "<B>[src]</B> has left the building..."
 			if(M_HARDCORE in mutations)
 				msg = "<B>[src]</B> whispers with \his final breath, <i>'i told u i was hardcore..'</i>"
+				if(miming)
```

Comment:
I double checked it and it worked fine, but the way you suggested does look a lot cleaner.

if(M_HARDCORE in mutations && !miming) works fine, right?